sys/src/9/ufs/ffs_vfsops.c

```diff
@@ -1591,7 +1591,7 @@ ffs_vgetf(MountPoint *mp, ino_t ino, int flags, vnode **vpp, int ffs_flags)
 		 * list by vput().
 		 */
 		free(buf);
-		releaseufsvnode(vp);
+		releasevnode(vp);
 		*vpp = nil;
 		return (error);
```

Comment:
why the ()?

Reply:
Seems it came with original. There was an age in what everybody used parenthesis in return statements.

Reply:
Ah, I was going to come back to this - I didn't follow the question.  If you're referring to the return statement, then Alvaro's right - it's the original code.